docs/cookbook.rst

```diff
@@ -124,7 +124,12 @@ Quick background about CumulusCI tasks
 
 All tasks in CumulusCI are python classes that subclass `cumulusci.core.tasks.BaseTask`.  The general usage of a task is two step: initialize an instance then call it to run the task.
 
-For most tasks, you'll want to override the `_run_task` method in your subclass to provide implementation.
+For most tasks, you'll want to override the `_run_task` method in your subclass to provide the implementation. The return value of this function is ignored. Exceptions from `cumulus.core.exceptions` should be raised to communicate task status to the user or flow. If no exceptions are thrown, the task is considered to have completed successfully.
```

Comment:
The return value is not necessarily ignored; it's stored as part of the StepResult when a task is running as part of a flow, where it could be looked up by tools that build on top of CumulusCI.

Reply:
I changed the wording from "The return value of this function is ignored" to "The return value of this function is saved as part of the StepResult.

I'm open to other suggestions if you have them. From the perspective of someone who had not written a brand new task from scratch before, the requirements of a task with respect to result codes, output, and exceptions weren't entirely clear.